app/gui/src/providers/openedProjects/projectStates.ts

```diff
@@ -319,6 +322,17 @@ export function useProjectStates() {
     details?: Ref<ProjectDetails>,
   ) {
     if (!backends.localBackend) return Err('Cannot open local project: Local Backend missing.')
+    await backends.localBackend
+      .startWatchingHybridProject(
+        info.id,
+        info.localProjectId,
+        info.parentId,
+        backends.remoteBackend.baseUrl,
+        httpClient.defaultHeaders,
+      )
+      .catch((err) => {
+        console.error(`Failed to start watching hybrid project ${info.id}`, err)
+      })
```

Comment:
We have EffectScope bound to the project here. Cleaning up on this scope dispose should be more resilient for changes. 

Let's move "stopWatchingHybridProject" from where it is now to "onScopeDispose" inside `scope` like this:
```suggestion
    await backends.localBackend
      .startWatchingHybridProject(
        info.id,
        info.localProjectId,
        info.parentId,
        backends.remoteBackend.baseUrl,
        httpClient.defaultHeaders,
      )
      .catch((err) => {
        console.error(`Failed to start watching hybrid project ${info.id}`, err)
      })
    scope.run(() => onScopeDispose(() => await backends.localBackend?.stopWatchingHybridProject(project.info.id).catch((err) => {
      console.error(`Failed to stop watching hybrid project ${project.info.id}`, err)
    })))
```
And then revert changes in `closeHybridProject`.